src/ui/component/doodle/index.js

```diff
@@ -0,0 +1,6 @@
+/* eslint react/display-name: 0 */
+
+import React from 'react';
+import 'css-doodle';
+
+export default ([rule = '']) => () => <css-doodle use="var(--rule)">{rule}</css-doodle>;
```

Comment:
Since we aren't using this component anywhere else, it can just be brought into the `splash` component. See my comment below.